_includes/code/howto/search.basics.py

```diff
@@ -532,7 +533,7 @@
 import weaviate.classes as wvc
 
 # Connect to the collection
-mt_collection = client.collections.get("AMultiTenancyCollection")
+mt_collection = client.collections.get("WineReviewMT")
```

Comment:
As above, consider something longer like WineReviewMulti

Reply:
Yup. Possibly later if we update the accompanying dataset.